Xamarin.Forms.Platform.iOS/Renderers/PageRenderer.cs

```diff
@@ -123,8 +123,9 @@ public void SetElementSize(Size size)
 
 		public override void LoadView()
 		{
+			//by default use the MainScreen Bounds so Effects can access the Container size
 			if (_pageContainer == null)
-				_pageContainer = new PageContainer(this);
+				_pageContainer = new PageContainer(this) { Frame = UIScreen.MainScreen.Bounds};
```

Comment:
I tested using the base.LoadView and see size created was the same as UIScreen.Bounds even inside a NavigationPage.